.goreleaser.yml

```diff
@@ -0,0 +1,22 @@
+build: 
+  ldflags: -s -w -X main.Version={{.Version}}
+  binary: trawl
+  env: 
+    - CGO_ENABLED=0 
+  goos: 
```

Comment:
Is there a reason Windows isn't included here?

Reply:
Other that I started based on a template for projects that do not have to run on Windows, no. Pushing an update.